### DISCUSSION TITLE
Development Environment

### DISCUSSION THREAD

**Discussion by lnbc1QWFyb24** [lightninglabs/lightning-node-connect#105] [Q&A] [answered]
Does anyone have a recommendation on a simple development environment for implementing and testing a LNC connection? For CLN commando connections I usually just dev on a regest node, but I see that LNC does not support regtest. I am trying to work out whether I should run a testnet or signet node in the cloud or locally for development, but curious if anyone has a better way of going about it. Would really love to be able to just simply use a local regtest node for dev.

**Comment by guggero** [comment, marked as answer]
LNC does support regtest. In fact, LNC is mostly network agnostic. The main difficulty is that on the server side you need Lightning Terminal which integrates some components that don't have a public server available (e.g. Loop and Pool). But I think with some of the latest patches you should be able to start Lightning Terminal as a whole even if some of the sub components complain about not being able to connect to a server.
So I think you _can_ use regtest.
Also, see this page that we use ourselves for testing: https://github.com/lightninglabs/lightning-node-connect/tree/master/example

The mailbox server address can be any hashmail server, independent of the network the LNC server/client are running on. So you can use the hard coded one or spin up your own [aperture](https://github.com/lightninglabs/aperture/blob/master/sample-conf.yaml#L208) instance.

**Reply by lnbc1QWFyb24** [reply]
Ok sweet I have got aperture, litd running and connecting to a Polar LND instance. Thanks for the info. Next step it to try getting LNC connected up to a local app.
